Microsoft.Toolkit.Graph/Microsoft.Toolkit.Graph.csproj

```diff
@@ -21,8 +21,8 @@
   </PropertyGroup>
```

Comment:
@azchohfi I meant to ask, I did notice that we multi-target here, but pretty sure we're just .NET Standard right, so do you remember why we might have done this?

Reply:
That is a great question... Git blame didn't help much:
https://github.com/windows-toolkit/Graph-Controls/commit/526138d0874e331fac0a90776fc032c8e3eb75f6#diff-30f6d4fe840f95e89a9fae72c74c5362
There is a failing build on PR #4, that got fixed by this commit, but the build itself is too old, and we purged the logs.

Reply:
I wonder if I made it to help with Uno or something. We can follow-up with @JeromeLaban later after UnoConf maybe?